### PR TITLE
Add ja.yml.  Update *.yml

### PR DESCRIPTION
**Pull request by shingo-nakanishi**

### CODE REVIEW

File: config/locales/zh-TW.yml [hunk 1]
```diff
@@ -59,6 +29,7 @@ zh-TW:
       help:
         redirect_uri: '每個網址必須只有一行'
         native_redirect_uri: '使用 %{native_redirect_uri} 作本地端測試'
+        scopes: 'Separate scopes with spaces. Leave blank to use the default scopes.'
```
Comment (tute) [inline config/locales/zh-TW.yml]: Let's leave the translations we don't have as blank. I believe that's encoded as `~`.

**Comment by tute** [comment]
Thank you! One comment for translations we don't have.

File: config/locales/zh-TW.yml (hunk 1 above)
Comment (shingo-nakanishi) [inline config/locales/zh-TW.yml]: thank you review.
you mean https://github.com/shingo-nakanishi/doorkeeper-i18n/commit/cdffebf042916f0f95aee773a0e5161f38cde09f (I made new commit)?

**Comment by tute** [comment]
> you mean shingo-nakanishi@cdffebf (I made new commit)?

I mean putting `~` instead of nothing, as explained in https://webtranslateit.com/docs/file_formats/yaml/.

**Comment by shingo-nakanishi** [comment]
thank you. I made new commit.

**Comment by tute** [comment]
Thank you!

**Pull request closed without merging**